jsonschema2pojo-cli/src/main/java/org/jsonschema2pojo/cli/Arguments.java

```diff
@@ -24,12 +24,7 @@
 import java.util.Iterator;
 import java.util.List;
 
-import org.jsonschema2pojo.AllFileFilter;
-import org.jsonschema2pojo.AnnotationStyle;
-import org.jsonschema2pojo.Annotator;
-import org.jsonschema2pojo.GenerationConfig;
-import org.jsonschema2pojo.NoopAnnotator;
-import org.jsonschema2pojo.SourceType;
+import org.jsonschema2pojo.*;
```

Comment:
This project doesn't use wildcard imports like this, could you remove these changes?